cluster-autoscaler/apis/go.mod

```diff
@@ -5,12 +5,12 @@ go 1.23.0
 toolchain go1.23.2
 
 require (
-	github.com/onsi/ginkgo/v2 v2.19.0
-	github.com/onsi/gomega v1.33.1
-	k8s.io/apimachinery v0.32.0-alpha.3
-	k8s.io/client-go v0.32.0-alpha.3
-	k8s.io/code-generator v0.32.0-alpha.3
-	sigs.k8s.io/structured-merge-diff/v4 v4.4.1
+	github.com/onsi/ginkgo/v2 v2.21.0
+	github.com/onsi/gomega v1.35.1
+	k8s.io/apimachinery v0.33.0-alpha.0
```

Comment:
wouldn't this tag be more appropriate?:

- https://github.com/kubernetes/kubernetes/tree/v1.32.0-rc.0

Reply:
It definitely would until we release 1.32 CA, but the `rc` tags aren't mirrored in the staged k8s.io repos for some reason (see https://github.com/kubernetes/api/tags), so `update-deps.sh` doesn't work with that tag.

`v1.33.0-alpha.0` should be okay though, if I'm looking correctly it only has an empty release commit on top of `v1.32.0-rc.0`:
* https://github.com/kubernetes/kubernetes/commits/v1.32.0-rc.0/
* https://github.com/kubernetes/kubernetes/commits/v1.33.0-alpha.0/

Reply:
sgtm